### PR TITLE
MyButton logo added to the projects section

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#17

### CODE REVIEW

**Comment by AHReccese** [comment]
We await Amir Boreshnavard's delivery of the dark mode logo with a transparent background.

File: src/Views/Pages/Home/Sections/Section3/index.tsx [hunk 1]
```diff
@@ -372,6 +374,18 @@ const projects = [
...
 		description: `DMeta is an open source Python package that removes metadata of Microsoft Office files.`,
 	},
+	{
+		title: 'MyButton',
+		repoName: 'mybutton',
+		dark_logo: mybuttonDark,
+		light_logo: mybuttonLight,
+		account: 'openscilab',
+		github: 'https://github.com/openscilab/mybutton',
+		webLink: 'https://github.com/openscilab/mybutton',
```
Comment (F-Shahali) [inline src/Views/Pages/Home/Sections/Section3/index.tsx:384]: I suggest to replace it with the URL of MyButton website: `https://mybutton.click`

**Review by F-Shahali — CHANGES_REQUESTED** [review]
Thanks for your effort.
Please take a look at my comment.
@AHReccese

File: src/Views/Pages/Home/Sections/Section3/index.tsx (hunk 1 above)
Comment (AHReccese) [reply src/Views/Pages/Home/Sections/Section3/index.tsx]: You are right, my bad.
Thanks

**Review by F-Shahali — APPROVED** [review]
LGTM

**Pull request merged**